design-toolkit/components/src/components/date-field/styles.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { tv } from 'tailwind-variants';
+
+export const DateFieldStyles = tv({
+  base: 'flex w-full gap-xs rounded-medium px-s py-xs font-display outline outline-interactive',
```

Comment:
Since you have multiple elements, this will become `slots`

And then you'll want to be sure to add a named group: https://github.com/gohypergiant/standard-toolkit/blob/main/design-toolkit/components/src/components/text-field/styles.ts#L17